var/spack/repos/builtin/packages/pika-algorithms/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+import sys
+
+from spack.package import *
+
+
+class PikaAlgorithms(CMakePackage):
+    """C++ parallel algorithms built on pika."""
+
+    homepage = "https://github.com/pika-org/pika-algorithms/"
+    url = "https://github.com/pika-org/pika-algorithms/archive/0.0.0.tar.gz"
+    git = "https://github.com/pika-org/pika-algorithms.git"
+    maintainers = ["msimberg", "albestro", "teonnik", "aurianer"]
+
+    version("0.1.0", sha256="64da008897dfa7373155595c46d2ce6b97a8a3cb5bea33ae7f2d1ff359f0d9b6")
+    version("main", branch="main")
+
+    generator = "Ninja"
+
+    map_cxxstd = lambda cxxstd: "2a" if cxxstd == "20" else cxxstd
+    cxxstds = ("17", "20")
+    variant(
+        "cxxstd",
+        default="17",
+        values=cxxstds,
+        description="Use the specified C++ standard when building",
+    )
+
+    # Build dependencies
+    depends_on("git", type="build")
+    depends_on("ninja", type="build")
+    depends_on("cmake@3.22:", type="build")
+
+    conflicts("%gcc@:8")
+    conflicts("%clang@:8")
+
+    # Other dependencies
+    depends_on("fmt@0.9:")
+    depends_on("pika@0.11:")
```

Comment:
I read from the project page that `fmt` and `boost` are required for tests and examples.


> The following are required for tests and examples:
> - [header-only Boost](https://boost.org/) 1.71.0 or greater
> - [fmt](https://fmt.dev/latest/index.html)

Probably you are not going to enable testing for the package at this stage but, does that mean that you can drop the `fmt` dependency (instead of adding also `boost`)?

Reply:
Good catch. Unfortunately I have to go the latter way (adding `boost`) since I've made the mistake of requiring the two even with tests and examples disabled. It's not such a big deal since they're anyway transitive dependencies required by pika, but we should fix that for the next release.

Reply:
Also for reference: https://github.com/pika-org/pika-algorithms/issues/9.